.github/workflows/package.yml

```diff
@@ -0,0 +1,112 @@
+# .github/workflows/build-wheels.yml
+name: Build and Package Wheels
+
+on:
+  pull_request:
+  push:
+
+env:
+  LIBRDKAFKA_VERSION: v2.6.1-gr
+
+jobs:
+
+  build-linux-x64:
+    name: Build wheels for Linux x64
+    runs-on: ubuntu-20.04
+    env:
+      OS_NAME: linux
+      ARCH: x64
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build wheels
+        run: |
+          ./tools/wheels/build-wheels.sh "${LIBRDKAFKA_VERSION#v}" wheelhouse
+          tar -czf wheelhouse-linux-${ARCH}.tgz wheelhouse
+          ls -al
+      - uses: actions/upload-artifact@v4
+        with:
+          name: wheels-${{ env.OS_NAME }}-${{ env.ARCH }}
+          path: wheelhouse/confluent_kafka*.whl
+
+  build-windows:
+    name: Build wheels for Windows
+    runs-on: windows-latest
+    env:
+      OS_NAME: windows
+      ARCH: x64
+      CHERE_INVOKING: yes
+      MSYSTEM: UCRT64
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup MSYS2
+        uses: msys2/setup-msys2@v2
+      - name: Build wheels
+        shell: bash
+        run: |
+          ./tools/mingw-w64/msys2-dependencies.sh
+          bash tools/mingw-w64/semaphore_commands.sh
+          bash tools/wheels/install-librdkafka.sh ${LIBRDKAFKA_VERSION#v} dest
+          tools/wheels/build-wheels.bat x64 win_amd64 dest wheelhouse
+          tar -czf wheelhouse-windows-${ARCH}.tgz wheelhouse
+      - uses: actions/upload-artifact@v4
+        with:
+          name: wheels-${{ env.OS_NAME }}-${{ env.ARCH }}
+          path: wheelhouse/confluent_kafka*.whl
+
+  package:
+    name: Package all wheels
+    needs: [build-linux-x64, build-windows]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          path: artifacts
+          pattern: wheels-*
+          merge-multiple: true
+      - name: Package wheels
+        run: |
+          cd artifacts
+          ls -al
+      - uses: actions/upload-artifact@v4
+        with:
+          name: confluent-kafka-python-wheels
+          path: artifacts/confluent_kafka-*.whl
+
+  publish_pypi_index:
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    name: Build a PyPI-compatible index
+    needs: package
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:    
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v4
+        with:
+          name: confluent-kafka-python-wheels
+          path: artifacts
+          
+      - name: Create release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+            artifacts/confluent_kafka*
```

Comment:
Attach wheels as release artifacts, these will be downloaded by the Python script to build a PIP-compatible simple package index.